Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml

```diff
@@ -555,8 +555,175 @@ tasks:
+    - label: GCP
+      condition:
+      - - operator: inList
+          left:
+            value:
+              complex:
+                root: inputs.cloudProvider
+            iscontext: true
+          right:
+            value:
+              simple: GCP,Google Cloud Platform,Google
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1240,
+          "y": 1000
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 25944d35-d446-47fe-84be-b282a9121908
+    type: regular
+    task:
+      id: 25944d35-d446-47fe-84be-b282a9121908
+      version: -1
+      name: Set tags grid field
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
+        `!GridFieldSetup keys=ip,src val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" gridfiled="gridfield"`
+      scriptName: GridFieldSetup
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      gridfield:
+        simple: asmtags
+      keys:
+        simple: key,value,source
+      val1:
+        complex:
+          root: PrismaCloud.Config.data.tags
+          accessor: key
+      val2:
+        complex:
+          root: PrismaCloud.Config.data.tags
+          accessor: value
+      val3:
+        simple: PrismaCloud
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1380,
+          "y": 1210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: f13478ac-13de-4b2f-87ca-e70b5a99438e
+    type: regular
+    task:
+      id: f13478ac-13de-4b2f-87ca-e70b5a99438e
+      version: -1
+      name: Set tags grid field
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
+        `!GridFieldSetup keys=ip,src val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" gridfiled="gridfield"`
+      scriptName: GridFieldSetup
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      gridfield:
+        simple: asmtags
+      keys:
+        simple: key,value,source
+      val1:
+        complex:
+          root: PrismaCloud.Config.data.tags
+          accessor: items
```

Comment:
Does GCP tags from Prisma, not populate `key` or `value`?

Reply:
right now for Prisma and GCP ${PrismaCloud.Config.data.tags} is network tags which are just a list and not k/v.  I want to create an enhancement to change this to labels instead which are k/v but it will be hard to do.